docs/conf/helpop-full.conf.example

```diff
@@ -992,30 +1011,25 @@ Note that all /STATS use is broadcast to online IRC operators.">
  A      Allows receipt of remote announcement messages.
  c      Allows receipt of local connect messages.
  C      Allows receipt of remote connect messages.
- d      Allows receipt of general (and sometimes random) debug
-        messages.
+ d      Allows receipt of general (and sometimes random) debug messages.
  f      Allows receipt of flooding notices.
- g      Allows receipt of globops (requires globops module).
- j      Allows receipt of channel creation notices (requires
-        chancreate module).
- J      Allows receipt of remote channel creation notices (requires
-        chancreate module).
+ g      Allows receipt of globops (requires the globops module).
+ j      Allows receipt of channel creation notices (requires the chancreate module).
+ J      Allows receipt of remote channel creation notices (requires the chancreate module).
  k      Allows receipt of local kill messages.
  K      Allows receipt of remote kill messages.
- l      Allows receipt of local linking related
-        messages.
- L      Allows receipt of remote linking related
-        messages.
- n      See local nickname changes (requires seenicks module).
- N      See remote nickname changes (requires seenicks modules).
- o      Allows receipt of oper-up, oper-down, and oper-failure
-        messages.
- O      Allows receipt of remote oper-up, oper-down, and oper-failure
-        messages.
+ l      Allows receipt of local linking related messages.
+ L      Allows receipt of remote linking related messages.
+ n      See local nickname changes (requires the seenicks module).
+ N      See remote nickname changes (requires the seenicks modules).
+ o      Allows receipt of oper-up, oper-down, and oper-failure messages.
+ O      Allows receipt of remote oper-up, oper-down, and oper-failure messages.
  q      Allows receipt of local quit messages.
  Q      Allows receipt of remote quit messages.
+ r      Allows receipt of local oper commands (requires the operlog module).
+ R      Allows receipt of remote oper commands (requires the operlog module).
  t      Allows receipt of attempts to use /STATS (local and remote).
- v      Allows receipt of oper-override notices (requires override module).
+ v      Allows receipt of oper-override notices (requires the override module).
  x      Allows receipt of local Xline notices (g/Z/q/k/e/R/shuns).
  X      Allows receipt of remote Xline notices (g/Z/q/k/e/R/shuns).">
 
```

Comment:
Keeping these all single line matches up with the `STATS` help above, with the `l` line being a few characters shorter than `J` here.